### ISSUE TITLE
The cursor in list result return should be actually called nextCursor

### ISSUE THREAD

**Issue by chrisli30** [AvaProtocol/EigenLayer-AVS#75]
### Problem
Currently it’s hard to detect the end of loop, using cursor for pagination.

#### Command to run:
https://github.com/AvaProtocol/ava-sdk-js/pull/45
`npx jest tests/getExecutions.test.ts -t "options.cursor works as pagination"` (this test need modification for this issue)

If the cursor points to the id of the last element, we will need a `hasMore: true` to indicate whether there are more, otherwise we can’t tell if there is more.

In the current implementation, the cursor returns "" at the end, so it’s the cursor of the next element, which doesn’t exsit. However, in client’s code, if they take "" and pass it back into the loop, **the list result will start over, retuning the 1st element of the list**. 

A working case would be, the current iteration takes the returned cursor value, and pass it into the next iteration, and the list retuns empty result. 

If we renames the `cursor` to `nextCursor`, empty string could be the indicator that we have reached the end of the list; however, we need to fix the empty string causing the list to start over problem. Basically, the beginning of the list should be distinctive from the end.

Could we refer to some standard design for the implementation of the cursor?

**Comment by v9n** [comment]
When the next cursor is empty we should stop.

Morallis does that

https://docs.moralis.com/web3-data-api/evm/reference/pagination 
> Every request will return a cursor that can be used to get the next result until there are no more results to return.


```
} while (cursor != "" && cursor != null);
```

We can do the same as well?

**Comment by chrisli30** [comment]
Yes, we can do the same, but could you give it a try, their end of list is a `null` or `""`?
